src/common/config/include/config.h.in

```diff
@@ -17,6 +17,9 @@ namespace config
         constexpr auto DEFAULT_RETRY_INTERVAL = @DEFAULT_RETRY_INTERVAL@;
         constexpr auto DEFAULT_BATCH_INTERVAL = @DEFAULT_BATCH_INTERVAL@;
         constexpr auto DEFAULT_BATCH_SIZE = @DEFAULT_BATCH_SIZE@;
+        constexpr auto QUEUE_DB_NAME = "@QUEUE_DB_NAME@";
```

Comment:
Why is this configurable?

Reply:
You're right, it doesn't make any sense to change the name by config. I'll move it to a header.